activitysim/defaults/tables/skims.py

```diff
@@ -39,3 +39,17 @@ def sovmd_skim(nonmotskm_matrix):
 def sovpm_skim(nonmotskm_matrix):
     # FIXME use the right omx file
     return skim.Skim(nonmotskm_matrix, offset=-1)
+
+
+@sim.injectable(cache=True)
+def skims():
+    skims = skim.Skims()
+    # FIXME - this is reusing the same skim as all the different kinds of skims
+    for typ in ["SOV_TIME", "SOVTOLL_TIME", "HOV2_TIME",
```

Comment:
Also - modes and variables w/in modes DO change and can be different for TOUR modes and TRIP modes.